tests/roles/glance_adoption/files/glance_cinder.yaml

```diff
@@ -6,7 +6,6 @@ spec:
     template:
       databaseInstance: openstack
       storage:
-        storageClass: "local-storage"
```

Comment:
@fmount Do you think it's ok to remove these?

Reply:
I mean, would it then just use what was in OpenStackControlPlane CR as the storageClass?

Reply:
Hey @jistr yes, I think the storageClass can be inherited by default from the top-level CR (OpenStackControlPlane), so it can be safely removed here!